docker/docker-compose-core.yml

```
@@ -37,7 +37,7 @@ services:
   # ICUBAM sms service
   app-sms:
     build:
-      context: "${ICUBAM_COMPOSE_CONTEXT:-.}"
+      context: "${ICUBAM_COMPOSE_CONTEXT:-..}"
```

See comment below about the mount
```suggestion
      context: "${ICUBAM_COMPOSE_CONTEXT:-.}"
```